cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -544,15 +569,16 @@ void ProcessShowDownloadedAction(const std::multimap<std::string, std::string> &
     const std::string id_to_reset(GetCGIParameterOrDefault(cgi_args, "reset"));
     if (not id_to_reset.empty()) {
         if (id_to_reset == "all")
-            ResetDeliveredRecordsForJournal(journal_id, db_connection);
+            ResetDeliveredRecordsForJournal(journal.id_, db_connection);
         else
             UpdateRecordDeliveryStateAndTimestamp(id_to_reset, ZoteroHarvester::Util::UploadTracker::DeliveryState::RESET, db_connection);
         at_least_one_action_done = "true";
     }
 
-    names_to_values_map.insertScalar("zeder_id", zeder_id);
-    names_to_values_map.insertScalar("zeder_instance", zeder_instance);
-    names_to_values_map.insertScalar("group", group);
+    names_to_values_map.insertScalar("id", std::to_string(journal.id_));
```

Comment:
I don't think you need to convert journal.id_ to a std::string.  Template::Map:: insertScalar is overloaded on the second parameter.